package.json

```diff
@@ -20,106 +20,105 @@
         "@octokit/rest": "^18.0.15",
-        "@octokit/types": "^6.6.0",
-        "@types/semver": "^7.3.5",
-        "axios": "^0.21.1",
-        "babel-loader": "^8.2.2",
-        "bootstrap": "^4.6.0",
+        "@octokit/types": "^6.34.0",
+        "@types/semver": "^7.3.9",
+        "axios": "0.21.4",
+        "bootstrap": "^4.6.1",
         "buffer": "^6.0.3",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-prettier": "^4.0.0",
         "fs-extra": "^9.1.0",
         "https-browserify": "^1.0.0",
         "jquery": "^3.5.1",
-        "openapi-client-axios": "3.13.1",
-        "openapi-client-axios-typegen": "3.13.1",
         "path-to-regexp": "^6.2.0",
         "popper.js": "^1.16.1",
         "process": "^0.11.10",
-        "react": "^17.0.1",
-        "react-bootstrap": "^1.4.3",
-        "react-intl": "^5.11.0",
+        "react": "^17.0.2",
+        "react-bootstrap": "^1.6.4",
+        "react-intl": "5.20.12",
         "react-json-view": "^1.21.3",
         "react-markdown": "^5.0.3",
-        "react-rnd": "^10.2.4",
-        "react-router-dom": "^5.2.0",
-        "react-transition-group": "^4.4.1",
+        "react-rnd": "^10.3.5",
+        "react-router-dom": "^5.3.0",
+        "react-transition-group": "^4.4.2",
         "semver": "^7.3.5",
         "stream-http": "^3.2.0",
-        "util": "^0.12.3"
-    },
-    "scripts": {
-        "generate_api": "npm run patch && npm run download_api && npm run generate_enums && npm run generate_exports && npm run typegen",
-        "postinstall": "npm run generate_api",
-        "start": "npx webpack serve --config webpack.dev.js",
-        "build": "npx webpack --config webpack.prod.js",
-        "msbuild": "npm run build && node scripts/msbuild.js",
-        "build-dev": "npx webpack --config webpack.dev.js",
-        "build-github": "npx webpack --config webpack.github.js",
-        "lint": "eslint ./src --max-warnings=0",
-        "lint-fix": "eslint --fix ./src --max-warnings=0",
-        "patch": "node scripts/patcher/patch.js",
-        "download_api": "node scripts/api_download.js",
-        "generate_enums": "node scripts/generate_enums.js > src/ApiClient/generatedcode/_enums.ts",
-        "generate_exports": "node scripts/generate_exports.js > src/ApiClient/generatedcode/schemas.d.ts",
-        "typegen": "npx typegen src/ApiClient/generatedcode/swagger.json > src/ApiClient/generatedcode/_generated.d.ts",
-        "test": "npm run lint && npm run build"
+        "util": "^0.12.4"
     },
     "devDependencies": {
-        "@babel/core": "^7.13.15",
-        "@babel/plugin-proposal-class-properties": "^7.13.0",
-        "@babel/preset-react": "^7.13.13",
-        "@babel/preset-typescript": "^7.13.0",
-        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1",
-        "@types/loadable__component": "^5.13.1",
-        "@types/react": "^16.14.2",
+        "@babel/core": "^7.16.0",
+        "@babel/plugin-proposal-class-properties": "^7.16.0",
```

Comment:
This plugin is now part of lalbel-env according to the docs

Reply:
but it's still required as a package

Reply:
Ok then